source/backend/api/Areas/Products/Controllers/ProductController.cs

```diff
@@ -0,0 +1,58 @@
+using MapsterMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Pims.Api.Models.Concepts;
+using Pims.Api.Policies;
+using Pims.Api.Services;
+using Pims.Dal.Security;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace Pims.Api.Areas.Projects.Controllers
+{
+    /// <summary>
+    /// ProductController class, provides endpoints for interacting with products.
+    /// </summary>
+    [Authorize]
+    [ApiController]
+    [ApiVersion("1.0")]
+    [Area("products")]
+    [Route("v{version:apiVersion}/[area]")]
+    [Route("[area]")]
+    public class ProductController : ControllerBase
+    {
+        #region fields
+        private readonly IProjectService _projectService;
+        private readonly IMapper _mapper;
+        #endregion
+
+        /// <summary>
+        /// Creates a new instance of a ProductController class, initializes it with the specified arguments.
+        /// </summary>
+        /// <param name="projectService"></param>
+        /// <param name="mapper"></param>
+        ///
+        public ProductController(IProjectService projectService, IMapper mapper)
+        {
+            _projectService = projectService;
+            _mapper = mapper;
+        }
+
+        /// <summary>
+        /// Retrieves a the acquisition file for the given product.
+        /// </summary>
+        /// <param name="productId"></param>
+        /// <returns>An array of contacts matching the filter.</returns>
+        [HttpGet("{productId}/acquisitionFiles")]
+        [HasPermission(Permissions.ProjectView)]
+        [Produces("application/json")]
+        [ProducesResponseType(typeof(AcquisitionFileModel), 200)]
+        [ProducesResponseType(typeof(Api.Models.ErrorResponseModel), 400)]
+        [SwaggerOperation(Tags = new[] { "product" })]
+        public IActionResult GetFiles(int productId)
```

Comment:
since this method appears to be specific to acq files you may want to change this method name, otherwise we may get collisions when we start adding other types of files.

Reply:
I think is a long. Will align the controller